CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKImage.java

```diff
@@ -242,21 +397,24 @@ public static CVKImage Create(  CVKDevice cvkDevice,
         cvkImage.tiling     = tiling;
         cvkImage.usage      = usage; 
         cvkImage.properties = properties;
-        cvkImage.aspectMask = aspectMask;
+        cvkImage.aspectMask = aspectMask;        
+        cvkImage.imageType  = height > 1 ? VK_IMAGE_TYPE_2D : VK_IMAGE_TYPE_1D;
+        cvkImage.viewType   = layers > 1 ? (height > 1 ? VK_IMAGE_VIEW_TYPE_2D_ARRAY : VK_IMAGE_VIEW_TYPE_1D_ARRAY) : (height > 1 ? VK_IMAGE_VIEW_TYPE_2D : VK_IMAGE_VIEW_TYPE_1D);
```

Comment:
nested ternarys are a bit confusing

Reply:
Noted, may change in future, but will keep amendments to functional issues for this change (otherwise I will never get back to work).